.env

```diff
@@ -20,4 +20,12 @@ REDIS_PORT_6379_TCP_PORT=
 
 SECRET_KEY_BASE=
 SIDEKIQ_CREDS=
-DEFAULT_PROXY_TARGET=
\ No newline at end of file
+DEFAULT_PROXY_TARGET=
+
+# Adobe Campaigns information and credentials
+ADOBE_ORG_ID=asdf
+ADOBE_ORG_NAME=cru
+ADOBE_TECH_ACCT=asdf
+ADOBE_API_KEY=asdf
```

Comment:
You're still planning on moving these (`ADOBE_API_KEY `, `ADOBE_API_SECRET`) to `destinations` table, correct?

Reply:
@hlbraddock that's correct.